src/explanations/pancakeswap.yaml

```diff
@@ -0,0 +1,113 @@
+
+
+    StableSwap is implemented using an automated market maker (AMM) model developed by Curve Finance, which utilizes liquidity pools to match buy and sell orders for a particular asset. The AMM model used by StableSwap includes a linear invariant constant sum curve in addition to the constant product formula used in normal AMMs. This helps to reduce impermanent loss, which occurs when the price of an asset changes while it is being held in a liquidity pool, and lowers the slippage compared to normal AMMs that use only the constant product formula.
+
+
+    StableSwap is designed to swap specific assets that are priced closely, such as USD stablecoins like HAY, BUSD, and USDT, or liquid staking tokens like stkBNB and aBNBc. When using StableSwap, users will pay lower trading fees compared to the standard 0.25% fee on normal PancakeSwap AMMs.
+
+    ### Lottery V2
+
+
+    Another popular feature of Pancakeswap is the lottery V2. It allows users to purchase lottery tickets using BSC tokens. The winners are chosen randomly, and the prizes can be anything from tokens to NFTs. The lottery V2 has become very popular on Pancakeswap, as it allows users to try their luck at winning a portion of the lottery's prize pool. The size of the prize pool depends on the number of tickets sold and the amount of BNB or CAKE that is wagered.
+
+
+    ### Prediction
+
+
+    Pancakeswap also has a prediction feature that allows users to make predictions on the future value of different tokens. The users who make the most accurate predictions are rewarded with tokens. This feature has become very popular on Pancakeswap as it allows users to test their knowledge of the cryptocurrency market and potentially earn some rewards.
+
+
+    ### Pottery
+    
+    
+    Pottery is a yield farming strategy on PancakeSwap that allows users to earn CAKE, the platform's native token, by depositing CAKE and participating in a lottery-style drawing. To participate in Pottery, users must deposit a minimum of 1 CAKE on the Pottery page during a specified period, which occurs on the Friday before the first Monday of each month. The deposited CAKE is directed to a lock-staking pool and locked for ten weeks. During this time, 80% of the total staking rewards generated from the deposited CAKE are directed to a prize pool for weekly drawings, and the remaining 20% are reserved for withdrawal after the ten weeks are complete. The chance of winning a prize in the weekly drawings is determined by the size of the user's deposit relative to the overall pool. Pottery uses Chainlink's implementation of VRF (Verifiable Random Function) to ensure the security and randomness of the drawing process. Pottery is a high-risk activity and can be volatile, so it is important for users to thoroughly research and understand the risks involved before participating.
```

Comment:
Please capture these details as it is very important

`With Pottery, you will be guaranteed to get back 100% of what you deposited in 10 weeks. However, you can only withdraw your CAKE deposit after 10 weeks of locking, without any other way to withdraw early.`

https://medium.com/pancakeswap/introducing-pancakeswap-pottery-lock-cake-before-aug-8-23-59-utc-5657f828253f